runtime/src/lib.rs

```diff
@@ -453,39 +452,33 @@ construct_runtime!(
 		UncheckedExtrinsic = UncheckedExtrinsic,
 	{
 		// System support stuff.
-		System: frame_system::{Pallet, Call, Config, Storage, Event<T>} = 0,
-		ParachainSystem: cumulus_pallet_parachain_system::{
-			Pallet, Call, Config, Storage, Inherent, Event<T>, ValidateUnsigned,
-		} = 1,
-		Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 2,
-		ParachainInfo: parachain_info::{Pallet, Storage, Config} = 3,
+		System: frame_system = 0,
+		ParachainSystem: cumulus_pallet_parachain_system = 1,
+		Timestamp: pallet_timestamp = 2,
+		ParachainInfo: parachain_info = 3,
```

Comment:
if im not mistaken, you don't have to specify indices explicitly; or maybe this is requirement for parachains to have strictly matching indices?

Reply:
Shouldn't matter, but there is a scary comment "Collator support. The order of these 4 are important and shall not change." (which seems strange to me), so I preferred not to touch the indices.

Reply:
I you think that it would be cleaner, then I may get rid of them